Can't walk dependency graph: Cannot find module

Using Ubuntu 22.04. Trialed installing locally, globally, with npm & yarn.

Reproducible error:
```bash
$ docker container run -it node bash

$ npm install -g jscad-now
npm WARN deprecated source-map-url@0.4.1: See https://github.com/lydell/source-map-url#deprecated
npm WARN deprecated resolve-url@0.2.1: https://github.com/lydell/resolve-url#deprecated
npm WARN deprecated sourcemap-codec@1.4.8: Please use @jridgewell/sourcemap-codec instead
npm WARN deprecated urix@0.1.0: Please see https://github.com/lydell/urix#deprecated
npm WARN deprecated source-map-resolve@0.5.3: See https://github.com/lydell/source-map-resolve#deprecated

added 351 packages in 8s

34 packages are looking for funding
  run `npm fund` for details

$ git clone https://github.com/danmarshall/jscad-now
Cloning into 'jscad-now'...
remote: Enumerating objects: 332, done.
remote: Counting objects: 100% (55/55), done.
remote: Compressing objects: 100% (46/46), done.
remote: Total 332 (delta 21), reused 22 (delta 9), pack-reused 277
Receiving objects: 100% (332/332), 193.91 KiB | 5.88 MiB/s, done.
Resolving deltas: 100% (157/157), done.

$ cd jscad-now/
$ jscad-now test/index.js 
[0000] info  Server running at http://127.0.0.1:8099/ (connect)
[0000] info  LiveReload running
press ENTER to close server and clean up "jscad-now.js" project file
Error: Can't walk dependency graph: Cannot find module 'test/index.js' from '/jscad-now/jscad-now.js'
    required by /jscad-now/jscad-now.js
    at /usr/local/lib/node_modules/jscad-now/node_modules/resolve/lib/async.js:146:35
    at processDirs (/usr/local/lib/node_modules/jscad-now/node_modules/resolve/lib/async.js:299:39)
    at isdir (/usr/local/lib/node_modules/jscad-now/node_modules/resolve/lib/async.js:306:32)
    at /usr/local/lib/node_modules/jscad-now/node_modules/resolve/lib/async.js:34:69
    at FSReqCallback.oncomplete (node:fs:199:21) {
  code: 'MODULE_NOT_FOUND',
  stream: [Labeled]
}

exiting, please wait...
```

--- a/src/server/index.js
+++ b/src/server/index.js
@@ -4,7 +4,9 @@ const createProject = require('./project')
 const createBudo = require('./budo')
 const listener = require('./listener')
 
-const modelFile = process.argv[2] || './index.js'
+const modelFile = process.argv.length > 2 
+  ? path.resolve(process.argv[2]) 
+  : path.resolve('./index.js')
 let modelFileExists = false
 
 try {

